lib/middleware/websocket/interactors/update_all_clients.rb

```diff
@@ -0,0 +1,21 @@
+require './lib/middleware/websocket/interactors/build_payload'
+
+module Websocket
+  module Interactor
+    class UpdateAllClients
+      def initialize(clients: clients)
+        @clients = clients
+        @payload = generate_payload
+        perform_update
+      end
+
+      def perform_update
+        @clients.each { |client| client.connection_client.write(@payload) }
+      end
+
+      def generate_payload
```

Comment:
Does this need to be a public method?

Reply:
@swcraig It doesn't, good call. Fixed now.